sqlglot/generator.py

```diff
@@ -258,6 +258,9 @@ class Generator:
     # INSERT OVERWRITE TABLE x override
     INSERT_OVERWRITE = " OVERWRITE TABLE"
 
+    # Dialect supports "SELECT INTO"
+    SUPPORT_SELECT_INTO = False
```

Comment:
Let's rename to `SUPPORTS_SELECT_INTO` which is more consistent with our naming schemes.

I'd also edit the comment a bit to have it say something like "Whether or not the SELECT .. INTO syntax is used instead of CTAS"